### ISSUE TITLE
Configurable bind addresses in datadog6

### ISSUE THREAD

**Issue by zoltan-toth-mw** [DataDog/datadog-agent#1513]
My host has multiple network interfaces, not just the loopback and eth0 and I need datadog 6 to listen on some of them. 
Right now datadog6 either listens only on localhost or on all interfaces, being open for the subnet that I don't want (let's forget firewalling).

In datadog5 there was a bind_host property which allowed me to change to a specific interface, though not to multiple interfaces, but still it was something.

**There should be a way in datadog6 to configure which addresses to bind.**
i.e.: bind_hosts: [ 127.0.0.1", "169.254.1.1" ]

**Comment by olivielpeau** [comment]
Hi @zoltan-toth-mw, thanks for the feedback!

We've just re-introduced the `bind_host` option in Agent 6 (see #1478), it will be included in the upcoming 6.1.0 release. The option is identical to Agent 5 and only allows binding to one interface.

Binding the Agent to multiple interfaces is something we could consider implementing, I'm going to open a feature request in our backlog for this. Let us know if you have more details on your use cases/how important the feature is for you.

**Comment by uded** [comment]
I would say a must have nowadays. Would simplify a lot IMHO in complex deployments and should be fairly simple to implement...

**Comment by daveadams** [comment]
Agreed, multiple bind_host values would allow a simpler networking configuration on some boxes we have with mixed container and host services.

**Comment by jbain** [comment]
This would be helpful for me as well. I'm having to work around this with some iptables magic at the moment

**Comment by deanhouseholder** [comment]
My problem was that I want to listen with both the "localhost" name AND the servers hostname (for other servers to talk to it).

My solution was to set the bind host to:
`bind_host: 0.0.0.0`

It now listens on localhost and on the hostname.

**Comment by daveadams** [comment]
@jacjtito Has this been addressed? It looks like the bind_host config element still only accepts one value.

**Comment by jacjtito** [comment]
Hey @daveadams , I closed this a bit fast after looking at the last comment, but I understand the original feature request is different. I'll reopen this issue and we will have someone look into it.
